types/rollup-plugin-add-git-msg/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for rollup-plugin-add-git-msg 1.1
+// Project: https://github.com/oplinjie/rollup-plugin-add-git-msg#readme
+// Definitions by: Jakub Jirutka <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/rollup-plugin-add-git-msg/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me